### ISSUE TITLE
[bug] Depth Map block failes to run, cannot find Cuda GPU on Arch Linux

### ISSUE THREAD

**Issue by ArcaneBlackwood** [alicevision/meshroom#880]
**Describe the bug**
Depth Map block failes to run, cannot find Cuda GPU on Arch Linux

**To Reproduce**
1. Download from git releases, extract.
2. Execute Meshroom
3. Add photographs using default config
4. Click "Start"
5. Error displayed in console

**Expected behavior**
The command `aliceVision_depthMapEstimation` to finish without an error, outputting its processed files

**Screenshots**

**Log**
Executed with command given by Meshroom in console, except verboseLevel modified to trace
```
Program called with the following parameters:
 * downscale = 2
 * exportIntermediateResults = 0
 * imagesFolder = "/home/arcane/Workspace/testScan/MeshroomCache/PrepareDenseScene/926b9eecfe443c2da00daa1e2c887fb6ee2cc4a7"
 * input = "/home/arcane/Workspace/testScan/MeshroomCache/StructureFromMotion/c8a5cb68f4bbd69c887662b089d00389fe31c896/sfm.abc"
 * maxViewAngle = 70
 * minViewAngle = 2
 * nbGPUs = 0
 * output = "/home/arcane/Workspace/testScan/MeshroomCache/DepthMap/59c2185a3a0c3b59c371fbd594421a71f5a0ad9d"
 * rangeSize = 3
 * rangeStart = 0
 * refineGammaC = 15.5
 * refineGammaP = 8
 * refineMaxTCams = 6
 * refineNDepthsToRefine = 31
 * refineNSamplesHalf = 150
 * refineNiters = 100
 * refineSigma = 15
 * refineUseTcOrRcPixSize = 0
 * refineWSH = 3
 * sgmGammaC = 5.5
 * sgmGammaP = 8
 * sgmMaxTCams = 10
 * sgmWSH = 4
 * verboseLevel = "trace"

[21:26:01.211155][warning] Could not determine number of CUDA cards in this system
[21:26:01.201495][warning] No CUDA-Enabled GPU.
[21:26:01.211177][error] cudaGetDeviceCount failed: unknown error
[21:26:01.211180][info] Can't find CUDA-Enabled GPU.
[21:26:01.211182][error] This program needs a CUDA-Enabled GPU (with at least compute capablility 2.0).
```

**Desktop (please complete the following and other pertinent information):**
 - OS: Arch linux 5.6.3-arch1-1
 - Python (packaged): Python 3.6.3
 - Meshroom version: Latest release on git
   - Binary version: 2019.2.0
 - GPU: NVIDIA Corporation TU106M [GeForce RTX 2060 Mobile]
 - `yay -Qs nvidia`
```
local/cuda 10.2.89-5
    NVIDIA's GPU programming toolkit
local/lib32-nvidia-utils 440.82-1
    NVIDIA drivers utilities (32-bit)
local/libvdpau 1.3-1
    Nvidia VDPAU library
local/libxnvctrl 440.82-1
    NVIDIA NV-CONTROL X extension
local/nvidia 440.82-11
    NVIDIA drivers for linux
local/nvidia-settings 440.82-1
    Tool for configuring the NVIDIA graphics driver
local/nvidia-utils 440.82-1
    NVIDIA drivers utilities
local/opencl-nvidia 440.82-1
    OpenCL implemention for NVIDIA
```

**Additional context**

**Comment by gryan315** [comment]
A couple questions: since you used yay to show Nvidia version, does that mean you installed from the aur? I would assume it's best to install the nvidia-dkms package from pacman. Do you have the CUDA package installed? I believe that's available in the pacman community repo. Since you're on a laptop, I'd also make sure optimus is working properly.

**Comment by ArcaneBlackwood** [comment]
I installed everything via pacman except for packages in the AUR.  I just used yay to list packages since I'm lazy and its quicker to type.  As listed, I have the package `cuda` installed.  I had not installed the nvidia-dkms package yet, Ive had bad experience with Nvidia and Xorg, so I am scared to install this package, I would like confirmation if I should.  Obviously I don't know much about this area, I apologize.

My setup doesn't use optimus, is this required?  Output from `glxinfo | grep NVIDIA`:
```
server glx vendor string: NVIDIA Corporation
client glx vendor string: NVIDIA Corporation
OpenGL vendor string: NVIDIA Corporation
OpenGL core profile version string: 4.6.0 NVIDIA 440.82
OpenGL core profile shading language version string: 4.60 NVIDIA
OpenGL version string: 4.6.0 NVIDIA 440.82
OpenGL shading language version string: 4.60 NVIDIA
OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 440.82
```
And in case, this is my Xorg config:
```
Section "ServerLayout"
	Identifier     "Layout0"
	Screen      0  "Screen0"
	InputDevice    "Keyboard0" "CoreKeyboard"
	InputDevice    "Mouse0" "CorePointer"
	Option "IndirectGLX" "on"
EndSection
Section "InputDevice"
	# generated from default
	Identifier     "Mouse0"
	Driver         "mouse"
	Option         "Protocol" "auto"
	Option         "Device" "/dev/psaux"
	Option         "Emulate3Buttons" "yes"
	Option         "ZAxisMapping" "4 5"
EndSection
Section "InputDevice"
	# generated from default
	Identifier     "Keyboard0"
	Driver         "kbd"
EndSection
Section "Monitor"
	Identifier     "Monitor0"
	VendorName     "Unknown"
	ModelName      "Unknown"
	#HorizSync       28.0 - 33.0
	#VertRefresh     43.0 - 72.0
	#Option         "DPMS"
EndSection
Section "Module"
    Load "modesetting"
EndSection
Section "Device"
	Identifier     "Device0"
	Driver         "nvidia"
	VendorName     "NVIDIA Corporation"
	BusID	"PCI:1:0:0"
EndSection
Section "Screen"
	Identifier     "Screen0"
	Device         "Device0"
	Monitor        "Monitor0"
	DefaultDepth    24
EndSection
```

**Comment by gryan315** [comment]
I'm not 100% familiar where Arch puts the nvidia and cuda binaries, but you could make sure they are available by adding their folders to your $PATH. On Clear Linux, the install path is /opt, so I run `export PATH=/opt/nvidia/bin:/opt/cuda/bin:$PATH` in my zshrc or bashrc.

**Comment by ArcaneBlackwood** [comment]
Cuda binaries are within my PATH variable, and NVIDIA binaries are in the typical /usr/bin location.  I checked by executing `nvidia-smi`, it returned with success, Also tried a Cuda command `nvvp` failed.  Apparently there were binaries in /usr/bin/ that were being ran instead, I deleted them and the Cuda commands now work.

Turns out that was the issue, now everything is seeming to work fine. Sorry for the trouble
